src/Controller/ErrorController.php

```diff
@@ -42,9 +42,8 @@ public function viewClasses(): array
      *
      * @param \Cake\Event\EventInterface<\Cake\Controller\Controller> $event Event.
      * @return \Cake\Http\Response|null|void
-     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint
      */
-    public function beforeRender(EventInterface $event)
+    public function beforeRender(EventInterface $event): void
```

Comment:
We can't add a `void` return types to any such "before" callback method as they can be overridden in the the child classes and have a return value.